Parsers/ASimAuthentication/Tests/Salesforce_ASimAuthentication_DataTest.csv

```diff
@@ -0,0 +1,24 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (33.33%) for field [ActorUserIdType] of type [Enumerated]: [""SaleforceId""] (Schema:Authentication)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (33.33%) for field [IpAddr] of type [IP Address]: [""Salesforce.com IP""] (Schema:Authentication)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (33.33%) for field [SrcIpAddr] of type [IP Address]: [""Salesforce.com IP""] (Schema:Authentication)"
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 30 records (100.0%) for field [EventProduct] of type [Enumerated]: [""Salesforce Service Cloud""] (Schema:Authentication)"
```

Comment:
Please add this value to Test case

Reply:
@anki-narravula I have changed EventProduct to 'Service Cloud'
EventVendor: Salesforce
EventProduct: Service Cloud
are already added in tester.csv